Library/Formula/brotli.rb

```diff
@@ -0,0 +1,23 @@
+class Brotli < Formula
+  desc "Generic-purpose lossless compression algorithm by Google."
+  homepage "https://github.com/google/brotli"
+  url "https://github.com/google/brotli/archive/v0.2.0.tar.gz"
+  sha256 "634d1089ee21b35e0ec5066cb5e44dd097e04e679e1e8c50bffa2b0dc77c2c29"
+
+  head "https://github.com/google/brotli.git"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  conflicts_with "bro", :because => "Both install a `bro` binary"
```

Comment:
Should be an empty line between the last `depends_on` and the first `conflicts_with`, Thanks!